app.js

```diff
@@ -1,17 +1,20 @@
 // import functions and grab DOM elements
+
 import { compareNumbers, getRandomNumber } from './utils.js';
 
 const button = document.getElementById('submit-button');
 const input = document.getElementById('number-input');
 const guessesRemaining = document.getElementById('guesses-remaining');
-
 const resultMessage = document.getElementById('result-message');
+
 // initialize state
-let guessCounter = 10;
+
+let guessCounter = 5;
 guessesRemaining.textContent = guessCounter; 
 
 const randomNumber = getRandomNumber();
```

Comment:
I like this idea of bringing in the getRandomNumber() as function. It's good practice. Good job.